Makefile

```
@@ -71,4 +74,18 @@ lint:
+
+.PHONY: audit
+audit:
+	@mkdir -p reports
+	$(PY) tools/audit_repo.py
+	$(PY) tools/dedupe_by_hash.py  # dry-run by default
+
+.PHONY: clean
+clean:
+	$(PY) tools/clean_repo.py --apply
+
+.PHONY: distclean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)